src/net/sourceforge/kolmafia/request/PlaceRequest.java

```diff
@@ -25,6 +31,8 @@ public class PlaceRequest extends GenericRequest {
   public static final TreeSet<String> places = new TreeSet<>();
   public boolean followRedirects = false;
 
+  private static Map<String, List<Integer>> aliasData = null;
```

Comment:
Just populate aliasData statically instead of ever initialising it as null? 

```java
private static Map<String, List<Integer>> nameAliases = Map.ofEntries(
  Map.entry("The Hippy Camp", List.of(...),
  ...
)
```

I gave the variable a better name there too

Reply:
Done.  Thank you.